components/Org/OrgDetail.js

```diff
@@ -107,70 +150,60 @@ const orgInstructionTab = (
   </span>
 )
 
+const orgOffersTab = (
+  <span style={{ fontSize: '1rem', fontWeight: 'bold', color: '#6549AA' }}>
+    <FormattedMessage
+      id='orgOffers'
+      defaultMessage='Offers'
+      description='show opportunities list on volunteer home page'
+    />
+  </span>
+)
+
+const orgSettingsTab = (
+  <span style={{ fontSize: '1rem', fontWeight: 'bold', color: '#6549AA' }}>
+    <FormattedMessage
+      id='orgSettings'
+      defaultMessage='Settings'
+      description='show opportunities list on volunteer home page'
+    />
+  </span>
+)
+
 const OrgDetail = ({ org, ...props }) => (
-  <FullPage>
+  <OrgPageWrapper>
     <Head>
       <title>Voluntarily - {org.name}</title>
     </Head>
     <PageHeaderContainer />
     <ProfileContainer>
       <ProfileImage src={org.imgUrl} alt={org.name} />
       <TitleContainer>
-        <H3Bold>{org.name}</H3Bold>
+        <OrgHeader>{org.name}</OrgHeader>
```

Comment:
How will OrgHeader be kept looking the same as the Activity and Opportunity Page headers?  All page headers should be the same hence the use of a common styled component.  
Consider refactoring.

Reply:
Currently the Activity and opportunity page headers are h1, I have now deleted OrgHeader and changed it to a h1 so its the same as the activity and opportunity page.